test/ibmq/test_ibmq_job.py

```diff
@@ -170,12 +168,6 @@ def test_run_multiple_device(self, backend):
         job_ids = [job.job_id() for job in job_array]
         self.assertEqual(sorted(job_ids), sorted(list(set(job_ids))))
 
-    def test_cancel(self):
-        """Test job cancellation."""
-        # Find the most busy backend
-        backend = most_busy_backend(self.provider)
-        submit_and_cancel(backend)
-
```

Comment:
Same question as above, why did you remove this test, and the other tests? They may need to be updated to not use qobj, but they're not dependent on qobj.